realsense_camera/launch/r200_nodelet_modify_params.launch

```diff
@@ -9,6 +9,8 @@
   <!-- These 'arg' tags are just place-holders for passing values from test files.
   The recommended way is to pass the values directly into the 'param' tags. -->
   <arg name="enable_depth"      default="true" />
+  <arg name="enable_ir"         default="true" />
```

Comment:
I had these as true to show how to override them to true. This was part of the request from Dave.